quizbizz/src/pages/authentication/signUp/SignupForm.jsx

```diff
@@ -132,7 +140,8 @@ const SignupForm = () => {
         <PrimaryButton
           buttonText="Create New Account"
           className="form-cta-button"
-          onClick={(e) => onSubmitForm(e)}
+          // onClick={(e) => onSubmitForm(e)}
```

Comment:
changes: can remove the commented code if it is not required.

Reply:
Done